test/integration/admin_dashboard_test.rb

```diff
@@ -34,4 +35,29 @@ class AdminDashboardTest < ActionDispatch::IntegrationTest
     refute page.has_content?('Welcome, Default_user!')
     assert page.has_content?("The page you were looking for doesn't exist.")
   end
+
+  test 'admin can send an email to all users with daily deal' do
+    User.create(username: 'MattR', password: 'password', email: 'matt@email.com')
+    User.create(username: 'Edgar', password: 'password', email: 'edgar@email.com')
+    User.create(username: 'MattS', password: 'password', email: 'stj@email.com')
+    create_items_associated_with_orders
+
+    admin = User.create(username: 'admin',
+                        password: 'password',
+                        role:      1)
+
+    visit login_path
+    fill_in 'Username', with: 'admin'
+    fill_in 'Password', with: 'password'
+    click_button 'Login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.